src/index.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import "./styles/global.css";
```

Comment:
Any reason as to why this is a double quotation?

Reply:
Prettier has been spazzing out on me...